### ISSUE TITLE
[CRITICAL BUG] cannot import libraries from scripts.

### ISSUE THREAD

**Issue by SmugNugg** [CCBlueX/LiquidBounce#2743]
### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

Latest newgen build (0.2.0)(night builds)

### Operating System

Windows 11

### Minecraft Version

1.20.4

### Describe the bug

I was developing a TP-Aura script when i encountered the problem. For some reason using https://maven.fabricmc.net/docs/yarn-1.20.4+build.3/index.html from the liquidbounce's scripting api page to get the latest net.minecraft.... libraries will result in a error saying "x cannot be accessed or has not been found.". 

### Steps to reproduce

- Make a new script
- Try for example "const TestLibrary = Java.Type("net.minecraft.block.entity");"
- Will result in an error.

### Client Log

```shell
don't have any
```

**Comment by superblaubeere27** [comment]
Yes. `Java.Type("net.minecraft.block.entity")` will not work, because it is a package, not a class ;)

Does `net.minecraft.client.MinecraftClient` work?

**Comment by SmugNugg** [comment]
> Yes. `Java.Type("net.minecraft.block.entity")` will not work, because it is a package, not a class ;)
> 
> Does `net.minecraft.client.MinecraftClient` work?

That was just an example . I also tried livingEntity and it too didn't work. I also went over to liquidbounce's source and took the .LivingEntity class from it's implemented library and yet no results.

**Comment by SmugNugg** [comment]
> Yes. `Java.Type("net.minecraft.block.entity")` will not work, because it is a package, not a class ;)
> 
> Does `net.minecraft.client.MinecraftClient` work?

oh sorry i was stupid going for the wrong ones. sorry for the waste of your time.

**Comment by SenkJu** [comment]
So does it work?

**Comment by SmugNugg** [comment]
> So does it work?

Yes
